web/util.py

```diff
@@ -0,0 +1,6 @@
+from flask import flash, redirect
+
+def flash_redirect(message: str | None = None, category = 'info', path = '/'):
```

Comment:
That is how I had it first,but something in my brain made me switch the order, I'll revert it because I agree that it probably makes more sense to do it that way and not provide a default

Also, pep8 is what put the spaces around the equals, so... :D

Reply:
✅